src/patternfly/layouts/Split/styles.scss

```diff
@@ -1,16 +1,27 @@
 @import "../../patternfly-utilities";
 
+$pf-l-split--gutter-size: $pf-global--gutter;
```

Comment:
same here, please follow the syntax for component variables.

Reply:
Done